task/sast-snyk-check/0.3/README.md

```diff
@@ -0,0 +1,39 @@
+# sast-snyk-check task
+
+## Description:
+
+The sast-snyk-check task uses Snyk Code tool to perform Static Application Security Testing (SAST) for Snyk, a popular cloud-native application security platform.
+
+Snyk's SAST tool uses a combination of static analysis and machine learning techniques to scan an application's source code for potential security vulnerabilities, including common issues such as SQL injection, cross-site scripting (XSS), and code injection attacks.
+
+> NOTE: This task is executed only if the user provides a Snyk token stored in a secret in their namespace. The name of the secret then needs to be supplied in the `snyk-secret` pipeline parameter.
+
+## Params:
+
+| name                     | description                                                                                                  |
+|--------------------------|--------------------------------------------------------------------------------------------------------------|
+| SNYK_SECRET              | Name of secret which contains Snyk token.                                                                    |
+| ARGS                     | Append arguments.                                                                                            |
+| SEVERITY_THRESHOLD       | Only vulnerabilities of the specified level or higher are reported (Possible values are low, medium or high) |
+| KFP_GIT_URL              | Link to the known-false-positives repository. If left blank, results won't be filtered                       |
+| PROJECT_NVR              | Name-Version-Release (NVR) of the scanned project, used to find path exclusions (it is optional)             |
+| RECORD_EXCLUDED          | File to store all excluded findings to (it is optional)                                                      |
```

Comment:
This should take `true` or `false`.  There is no reason to configure the file name for `RECORD_EXCLUDED` if the the default list of (not excluded) findings is not configurable.  Let's stick to `excluded-findings.json` for consistency across the SAST scanning CI tasks.